content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,321 @@
+      :align: center
+      :alt: The Save as Favorite Filter pop-up can save the lost leads criteria for later.
+
+8. Once all the components of the email campaign are complete:
+
+   - Click :guilabel:`Send` to immediately send the email;
+
+     OR
+   - Click :guilabel:`Schedule` to send the email at a future date and time.
+
+.. tip::
+   Consider using A/B Testing to send an alternate version of the email to a percentage of the
+   target leads. This can help determine what subject lines and body content produce the best
+   click-through rates before sending a final version to the remaining leads.
+
+   To do so, open the :guilabel:`A/B Tests` tab and check the box next to :guilabel:`Allow A/B
+   Testing`. Adjust the parameters as needed, and click :guilabel:`Create an Alternative Version`.
+
+   .. image:: lost_leads_email/ab-testing.png
+      :align: center
+      :alt: The A/B Tests tab with the Allow A/B Testing box checked to create an alternate version.
+
+.. _email_marketing/select_lost_reasons:
+
+Select appropriate lost reasons
+-------------------------------
+
+When a lead is marked as lost, Odoo recommends selecting a *Lost Reason* to indicate why the
+opportunity did not result in a sale. Doing so keeps the pipeline organized and reporting data
+accurate, and also generates potential to follow up with the lead in the future.
+
+If an existing *Lost Reason* is not applicable, users with the necessary permissions can create new
+ones, which means the lost reasons in a database vary from organization to organization and pipeline
+to pipeline. For more information on *Lost Reasons*, including the creation of them, refer to
+:doc:`../../sales/crm/pipeline/lost_opportunities`.
+
+By default, Odoo does include a few common reasons such as:
+
+- *Too expensive*
+- *We don't have people/skills*
+- *Not enough stock*
+
+When determining which reasons to include in a lost leads reactivation email, consider what the
+email is advertising in order pinpoint one or more relevant lost reasons. Then add a rule stating
+:guilabel:`Lost Reason` *does not contain* `_____` for every reason in the database **except** for
+the relevant one(s).
+
+.. example::
+   If the email advertises a selection of previously limited merchandise that is now back in stock,
+   it makes sense to target leads with the lost reason: *not enough stock*. To do so, add rules that
+   exclude all reasons except *not enough stock*.
+
+   .. image:: lost_leads_email/out-of-stock.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons except for Out of Stock.
+
+   If the email advertises a price reduction, it makes sense to target leads with the lost reason:
+   *too expensive*. To do so, add rules that exclude all reasons except *too expensive*.
+
+   .. image:: lost_leads_email/too-expensive.png
+      :align: center
+      :alt: A list of filter rules that exclude all lost reasons except for Too Expensive.
```

Comment:
Can you add the filter 'does contain' too expensive or not enough stock? OR does it need to be the other way around and include the does not contain filters?

Reply:
Yea that's better logic actually, to do the *inclusive* option instead of hunting down and including all of the exclusions 👍